packages/react/src/utils/useScrollLock.ts

```diff
@@ -167,7 +167,9 @@ function preventScrollInsetScrollbars(referenceElement: Element | null) {
   return () => {
     resizeFrame.cancel();
     cleanup();
-    win.removeEventListener('resize', handleResize);
+    if (win && win.removeEventListener) {
+      win.removeEventListener('resize', handleResize);
+    }
```

Comment:
We should add a comment to explain why the check is necessary. And iiuc, only `win.removeEventListener` should be checked.